app/src/main/java/org/xbmc/kore/service/library/SyncMovies.java

```diff
@@ -159,11 +159,6 @@ public void onSuccess(List<VideoType.DetailsMovie> result) {
 
                 LogUtils.LOGD(TAG, "syncAllMovies, movies gotten: " + result.size());
                 if (result.size() == LIMIT_SYNC_MOVIES) {
-                    // Max limit returned, there may be some more movies
-                    // As we're going to recurse, these result objects can add up, so
-                    // let's help the GC and indicate that we don't need this memory
-                    // (hopefully this works)
-                    result = null;
```

Comment:
Unless you can clearly explain that setting this to null is useless I suggest we keep it.